### ISSUE TITLE
Reviews: add first-time hints to explain the + and check mark buttons

### ISSUE THREAD

**Issue by davidsansome** [davidsansome/tsurukame#22]

**Comment by turtlemaster19** [comment]
Good idea, but I think this should stay right where it is at the bottom, since it can easily be figured out by a first time user by tapping on them to see what happens.

**Comment by turtlemaster19** [comment]
I think the checkmark is self-explanatory, but I agree that the plus button could use some sort of explaining. Maybe a first-time tutorial with just a little information might be in order, available at the login screen, perhaps?

**Comment by turtlemaster19** [comment]
@davidsansome Is this necessary?

The checkmark button is self-explanatory, it moves on just like in WK.

The plus button is only enabled if the setting "Ignore Typos & Add Synonyms" is turned on by the user, in which case they would know exactly what the button does.

I haven't seen any comments on the forums ever asking what these buttons do. With all due respect, can we close this issue?

**Comment by smearedink** [comment]
> @davidsansome Is this necessary?
> 
> The checkmark button is self-explanatory, it moves on just like in WK.
> 
> The plus button is only enabled if the setting "Ignore Typos & Add Synonyms" is turned on by the user, in which case they would know exactly what the button does.
> 
> I haven't seen any comments on the forums ever asking what these buttons do. With all due respect, can we close this issue?

I actually have found the “+” a little confusing, and even after months of using this very nice app, I’m not 100% sure what it does because as far as I can see there’s no explanation for it.

Going from memory (as I don’t have any reviews at this moment), it lets you choose “my answer was correct” or “show me this item again”. It took me a little time to discover what was under the “+” at all since I didn’t want to hit random buttons, and then I was a bit afraid to try using the first option for awhile, though it seems to do what I want/expect. I occasionally become afraid it is also adding my answer as a synonym but... I don’t think it is?

I’m still not entirely clear on the second item. Does it mark you correct but bring it back soon after? Does it act like you didn’t do it at all? I’ve never been sure.

So in my opinion, these and some other UI elements could definitely benefit from in-app documentation.

**Comment by UInt2048** [comment]
@smearedink Objection noted. I will retract my previous comment.

For documentation purposes:

My answer was correct = Override (pretend you answered one of the meanings)

Ask again later = Pretend this review never happened, and reshuffle it with the others.

Add synonym = Tell WK the exact input you typed should always be accepted, even on the site

**Comment by smearedink** [comment]
Thanks! :) I don’t usually get the “add a synonym” option, actually, just the other two. I saw “add a synonym” a day or two ago though. Is there some condition for it appearing?

**Comment by smearedink** [comment]
Hang on I just realized it’s obviously for when it’s a meaning, not a reading. Duh. Don’t mind me!
